Include/pegen_interface.h

```diff
@@ -11,6 +11,8 @@ extern "C" {
 PyAPI_FUNC(mod_ty) PyPegen_ASTFromFile(const char *filename, int mode, PyArena *arena);
 PyAPI_FUNC(mod_ty) PyPegen_ASTFromString(const char *str, int mode, PyCompilerFlags *flags,
                                          PyArena *arena);
```

Comment:
Perhaps off-topic: comparing this to `PyParser_ASTFromString()`, the latter has a `char *filename` argument (before `int mode`) -- maybe for consistency we should add it here too? (But not until we've dealt with all the other fallout from #96).